dev/fattest.simplicity/src/componenttest/rules/repeater/EE7FeatureReplacementAction.java

```diff
@@ -27,6 +27,7 @@
                                                  "javaMail-1.5",
                                                  "cdi-1.2",
                                                  "jca-1.7",
+                                                 "jcaInboundSecurity-1.0",
```

Comment:
Shouldn't this same change go into EE6FeatureReplacementAction as well?

Reply:
Unless I use that action, I didn't plan to update it, as the ee6 version is missing a lot of the features. For example, JCA is not even in there.